omnibus_overrides.rb

```diff
@@ -7,9 +7,11 @@
 override :rubygems, version: "3.0.2"
 override :bundler, version: '~> 1.17'
 override :'omnibus-ctl', version: "master"
-override :chef, version: "v14.11.21"
-override :ohai, version: "v14.8.11"
+override :chef, version: "v15.0.269"
```

Comment:
If we're using Chef 15 I think we should use ruby 2.6.3, rubygems and bundler 1.17.2 3.0.3 based on the [deps that ship with our chef omnibus packages](https://github.com/chef/chef/blob/d810b363da8ccdbed92ce92419d6c6e7f2f0835e/omnibus_overrides.rb)

Reply:
Fixed